pkg/kubelet/kubelet.go

```diff
@@ -320,7 +320,8 @@ func NewMainKubelet(
 		resolverConfig:                 resolverConfig,
 		cpuCFSQuota:                    cpuCFSQuota,
 		daemonEndpoints:                daemonEndpoints,
-		containerManager:               containerManager,
+		clock:                          util.RealClock{},
+		outOfDiskTransitionFrequency: outOfDiskTransitionFrequency,
```

Comment:
I think this is a gofmt bug. Since gofmt is run every time I save this file in the editor, this strange alignment keeps coming back. I even tried opening this file in an editor without gofmt integration, manually adjusting the alignment and then running gofmt on the command line again. This unaligned formatting just keeps happening.